chain-signatures/contract/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mpc-contract"
-version = "1.0.0-rc.5"
+version = "1.0.0"
```

Comment:
are we planning on locking the contract yet? If not, we should still have it be in RC

Reply:
We need to do that someday. We can always change the major version.